koans/AboutApplyingWhatWeHaveLearnt.js

```diff
@@ -75,15 +84,17 @@ describe("About Applying What We Have Learnt", function() {
         }
     }
 
-    expect(ingredientCount['mushrooms']).toBe(FILL_ME_IN);
+    expect(ingredientCount['mushrooms']).toBe(2);
   });
 
   it("should count the ingredient occurrence (functional)", function () {
-    var ingredientCount = { "{ingredient name}": 0 };
+    var ingredientCount = _(products).chain().pluck('ingredients').flatten();
```

Comment:
Once flattened you'll need to reduce this into the required `ingredientCount` object.

Let me know if we need to workshop this.